CHANGELOG.md

```diff
@@ -2,9 +2,19 @@
 
 ## v73.0.0-SNAPSHOT - unreleased
 
+### 💥 Breaking Changes (upgrade difficulty: 🟢 LOW - Hoist core update)
+
+* Requires `hoist-core >= 28.1` with new APIs to support JSON searching in the Admin Console.
```

Comment:
We should ensure this is the HC version we use for the next release - I think it likely can be.